feat: Refactor Enable And Disable Module Commands (#1918)
kyma-project/cli: docs/gen-docs/kyma_alpha_delete_module.md

---
title: kyma alpha delete module
---

> Delete a module in the cluster or in the given Kyma resource.

Reviewer: ```suggestion
Deletes a module from the cluster or from the given Kyma resource.
```

## See also

> * [kyma alpha delete](kyma_alpha_delete.md)	 - Disables a resource in the Kyma cluster.

Reviewer: ```suggestion
* [kyma alpha delete](kyma_alpha_delete.md)	 - Deletes a resource from the Kyma cluster.
```
